Mathlib/FieldTheory/RatFunc.lean

```diff
@@ -1694,11 +1697,19 @@ theorem coe_apply : coeAlgHom F f = f :=
   rfl
 #align ratfunc.coe_apply RatFunc.coe_apply
 
+theorem coe_coe (P : Polynomial F) : (P : LaurentSeries F) = (P : RatFunc F) := by
+  erw [RatFunc.coe_def, RatFunc.coeAlgHom, liftAlgHom_apply, RatFunc.num_algebraMap,
+    RatFunc.denom_algebraMap P, map_one, div_one]
+  rfl
+
 @[simp, norm_cast]
 theorem coe_zero : ((0 : RatFunc F) : LaurentSeries F) = 0 :=
   (coeAlgHom F).map_zero
 #align ratfunc.coe_zero RatFunc.coe_zero
 
+theorem coe_ne_zero {f : Polynomial F} : f ≠ 0 → (↑f : PowerSeries F) ≠ 0 := by
```

Comment:
Is there a reason `f \neq 0` is not a variable?

Reply:
Basically that I never remember what is prefered. I have changed it (is it better?)